### ISSUE TITLE
Ability to use with Tensorflow.js

### ISSUE THREAD

**Issue by Riz-waan** [uber/ludwig#193]
**Describe the use case**
You could run all the models in the web and create predictions using the clients resources. Such as for like chatbots and stuff.

**Describe the solution you'd like**
The current library ported to tensorflow.js

**Describe alternatives you've considered**
Have this run on the backend and interact with a webserver.

**Comment by w4nderlust** [comment]
I haven't explored much TensorFlow.js yet. I will explore if there is at least a way to lead models you trained with regular tensorflow with the js version. Considering this as a feature request.

**Comment by Riz-waan** [comment]
Yes, many models trained on tensor flow can be run on tensorflow.js

On Mar 11, 2019 2:06 AM, Piero Molino <notifications@github.com> wrote:

I haven't explored much TensorFlow.js yet. I will explore if there is at least a way to lead models you trained with regular tensorflow with the js version. Considering this as a feature request.

**Comment by w4nderlust** [comment]
Good to know, added to the backlog!

**Comment by FineonX** [comment]
Any news on that Enhancement?

**Comment by slavakurilyak** [comment]
Any updates on this? I am very interested in seeing Ludwig democratize machine learning for the web and beyond

**Comment by w4nderlust** [comment]
@slavakurilyak  unfortunately no, we haven't got to it yet. We would welcome contributions from the community as our backlog is pretty full at the moment
